js/player.js

```diff
@@ -1,55 +0,0 @@
-function playerTank() {
-  player.push({
```

Comment:
I like that player is an array because then it would be easy to add multiple players - something like "press space" to clone etc. but just for practice -> try re-writing it without the array but with a variable -> player. 

also, since this is an array - aptly name it players!